lib/controllers/books.js

```diff
@@ -0,0 +1,11 @@
+const { Router } = require('express');
+const Book = require('../models/Book');
+
+module.exports = Router()
+  .get('/', async (req, res) => {
+    const books = await Book.getAll();
+    const ids = books.map((book) => ({ id: book.id, title: book.title, released: book.released }));
```

Comment:
There is no need for a map here, because the data returned from 
```
const books = await Book.getAll();
```
is already returning the correct values, so you can pass books directly to your res.json(books).